src/components/TodoBar/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
+import ActionDelete from 'material-ui/svg-icons/action/delete';
+import { Toolbar, ToolbarGroup } from 'material-ui/Toolbar';
+import RaisedButton from 'material-ui/RaisedButton';
+
+const styles = {
+  toolbarGroup: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  radioButtonGroup: {
+    width: '60%',
+    display: 'flex',
+  },
+  radioButton: {
+    width: '33%',
+  }
+}
+
+const TodoBar = ({ deleteCompleted, filterComplete, filterUncomplete, handleFilterChange }) => {
+  return (
+    <Toolbar style={styles.toolbar}>
+      <ToolbarGroup style={styles.toolbarGroup}>
+        <RadioButtonGroup name="todoFilter" defaultSelected="none" onChange={(e) => handleFilterChange(e)} style={styles.radioButtonGroup}>
+          <RadioButton
+            value="none"
+            label="Show All"
+            style={styles.radioButton}
+          />
+          <RadioButton
+            value="uncomplete"
```

Comment:
'uncomplete' should be 'incomplete'. You'll need to do a find and replace, because I'm sure this is used elsewhere in your project.